### PR TITLE
Update artifactid  and readme

### PR DESCRIPTION
**Pull request by wujack778**
Update artifactId, groupId and readme
to address issue #11374 and issue #11153

### CODE REVIEW

**Review by saragluna — APPROVED** [review]
LGTM.

File: sdk/spring/azure-spring-boot-starter-active-directory/README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ With Spring Starter for Azure Active Directory, now you can get started quickly
 [//]: # "{x-version-update-start;com.microsoft.azure:azure-active-directory-spring-boot-starter;current}"
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot-starter-active-directory/README.md:19]: This line should also update to com.azure:azure-spring-boot-starter-active-directory, same as other README.md files

File: sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ application to use the Spring annotations and classes to protect the web app.
    ```xml
    <dependency>
        <groupId>com.azure</groupId>
-       <artifactId>azure-active-directory-b2c-spring-boot-starter</artifactId>
+       <artifactId>azure--spring-boot-starter-active-directory-b2c</artifactId>
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md]: `azure--spring`, delete '-' here

File: sdk/spring/azure-spring-boot-starter/README.md [hunk 3]
```diff
@@ -13,9 +13,9 @@ This repository is for Spring Boot Starters of Azure services. It helps Spring B
 [//]: # ({x-version-update-start;com.microsoft.azure:azure-spring-boot-starter;current})
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot-starter/README.md:13]: same here

File: sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md [hunk 4]
```diff
@@ -14,12 +14,12 @@ while protecting the identities of your customers at the same time.
 - [Maven](http://maven.apache.org/) 3.0 and above
 
 ### Include the package
-[//]: # "{x-version-update-start;com.microsoft.azure:azure-active-directory-b2c-spring-boot-starter;current}"
+[//]: # "{x-version-update-start;com.azure:azure-active-directory-b2c-spring-boot-starter;current}"
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md:17]: same here

File: sdk/spring/azure-spring-boot-starter-cosmosdb/README.md [hunk 5]
```diff
@@ -11,12 +11,12 @@
 - [Maven](http://maven.apache.org/) 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.microsoft.azure:azure-cosmosdb-spring-boot-starter;current})
+[//]: # ({x-version-update-start;com.azure:azure-cosmos-spring-boot-starter;current})
 ```xml
```
Comment (zhoufenqin) [inline sdk/spring/azure-spring-boot-starter-cosmosdb/README.md:15]: here

**Review by zhoufenqin — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-cosmosdb/README.md (hunk 5 above)
Comment (wujack778) [reply sdk/spring/azure-spring-boot-starter-cosmosdb/README.md:15]: all right

**Review by zhoufenqin — APPROVED** [review]

**Pull request merged**